.github/workflows/release.yaml

```diff
@@ -0,0 +1,28 @@
+# Create a new release on a pushed tag
+# Mostly taken from https://github.com/bazel-contrib/rules-template/blob/main/.github/workflows/release.yaml
+name: Release
+on:
+  # For now, only trigger when pushing a new tag, or manually from the GitHub UI.
+  push:
+    tags:
+      - "v*.*.*"
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: git tag to release. The tag must exist.
+        required: true
+        type: string
+jobs:
+  release:
+    uses: bazel-contrib/.github/.github/workflows/release_ruleset.yaml@v7.2.2
+    with:
+      release_files: gcc-toolchain-*.tar.gz
+      prerelease: false
```

Comment:
```suggestion
      prerelease: true
```

Reply:
This will tag the releases as "pre-releases". Are we sure that's what we want?

Reply:
That's the current state of the repository. We are < 1.0.0.